libs/application/templates/financial-aid/src/lib/messages/confirmation.ts

```diff
@@ -0,0 +1,99 @@
+import { defineMessages } from 'react-intl'
+
+export const confirmation = {
+  general: defineMessages({
+    sectionTitle: {
+      id: 'fa.application:section.confirmation.general.sectionTitle',
+      defaultMessage: 'Staðfesting',
+      description: 'Confirmation section title',
+    },
+    pageTitle: {
+      id: 'fa.application:section.confirmation.general.pageTitle',
+      defaultMessage: 'Staðfesting',
+      description: 'Confirmation page title',
+    },
+  }),
+  nextSteps: defineMessages({
+    title: {
+      id: 'fa.application:section.confirmation.nextSteps.title',
+      defaultMessage: '   Hér eru næstu skref',
+      description: 'Confirmation next step title',
+    },
+    content: {
+      id: 'fa.application:section.confirmation.nextSteps.content#markup',
+      defaultMessage:
+        '* Vinnsluaðili sveitarfélagsins vinnur úr umsókninni. Umsóknin verður afgreidd eins fljótt og auðið er. \n* Ef umsóknin er samþykkt getur þú reiknað með útgreiðslu í byrjun október. \n* Ef þörf er á frekari upplýsingum eða gögnum til að vinna úr umsókninni mun vinnsluaðili sveitarfélagsins hafa samband.',
+      description: 'Confirmation next steps',
+    },
+    contentWithSpouse: {
+      id: 'fa.application:section.confirmation.nextSteps.content#markup',
+      defaultMessage:
+        '* Þú og maki þinn hlaðið upp nauðsynlegum gögnum sem þarf til úrvinnslu umsóknar. \n* Vinnsluaðili sveitarfélagsins vinnur úr umsókninni. Umsóknin verður afgreidd eins fljótt og auðið er. \n* Ef umsóknin er samþykkt getur þú reiknað með útgreiðslu í byrjun október. \n* Ef þörf er á frekari upplýsingum eða gögnum til að vinna úr umsókninni mun vinnsluaðili sveitarfélagsins hafa samband.',
+      description: 'Confirmation next steps',
+    },
+  }),
+  links: defineMessages({
+    title: {
+      id: 'fa.application:section.confirmation.links.title',
+      defaultMessage: 'Frekari aðgerðir í boði',
+      description: 'Confirmation links title',
+    },
+    statusPage: {
+      id: 'fa.application:section.confirmation.links.statusPage',
+      defaultMessage: 'Sjá stöðu umsóknar',
+      description: 'Confirmation link to status page',
+    },
+    municipalityInfoPage: {
+      id: 'fa.application:section.confirmation.links.muncipalityInfoPage',
+      defaultMessage: 'Upplýsingar um fjárhagsaðstoð hjá sveitarfélaginu',
+      description: 'Confirmation link to status page',
+    },
```

Comment:
Should this be markdown and then have both strings together? In case we add another string or remove one?

Reply:
We should also add the hrefs here

Reply:
Yeah I thought so too, but then we cant have icon for the links like in figma? Change the design or the code ? 🙃